src/App.tsx

```diff
@@ -1,9 +1,22 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [searchQuery, setSearchQuery] = useState('');
+
+  const handleSearch = (event: { target:
+  { value: React.SetStateAction<string>; }; }) => {
+    setSearchQuery(event.target.value);
+  };
+
+  const [movies] = useState(moviesFromServer);
```

Comment:
What is the purpose of this state? Why not passing `moviesFromServer` to filteredMovies directly?